.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.analysis.extraPaths": ["./privacy_evaluator"]
+}
```

Comment:
The .vscode directory should probably not be part of the pull request, right?

Reply:
correct, just remove it!